pyproject.toml

```diff
@@ -1,14 +1,13 @@
 [build-system]
 requires = ["maturin>=1.0,<2.0"]
-
 build-backend = "maturin"
 
 [project]
 name = "ruff"
 version = "0.0.291"
 description = "An extremely fast Python linter, written in Rust."
-authors = [{ name = "Charlie Marsh", email = "charlie.r.marsh@gmail.com" }]
-maintainers = [{ name = "Charlie Marsh", email = "charlie.r.marsh@gmail.com" }]
+authors = [{ name = "Astral", email = "hey@astral.sh" }]
+maintainers = [{ name = "Astral", email = "hey@astral.sh" }]
```

Comment:
Should this be "Astral Software Inc."? I'm not sure. I see Prefect uses the full company name.

Reply:
Sure Dagster uses "Dagster Labs" and Django uses "Django Software Foundation"

https://github.com/dagster-io/dagster/blob/master/python_modules/dagster/setup.py#L38
https://github.com/django/django/blob/main/setup.cfg#L5

Reply:
Also https://discuss.python.org/t/the-author-maintainer-distinction-problem-and-pep-621/4562/21 should only maintain this once